flutter_local_notifications/README.md

```diff
@@ -43,37 +43,208 @@ A cross platform plugin for displaying local notifications.
 * [iOS] Request notification permissions and customise the permissions being requested around displaying notifications
 * [iOS] Display notifications with attachments
 
-Note that this plugin aims to provide abstractions for all platforms as opposed to having methods that only work on specific platforms. However, each method allows passing in "platform-specifics" that contains data that is specific for customising notifications on each platform. This approach means that some scenarios may not be covered by the plugin. Developers can either fork or maintain their code for showing notifications in these situations. Note that the plugin still under development so expect the API surface to change over time.
 
-**IMPORTANT NOTES**:
 
-* Recurring notifications on Android use the [Alarm Manager](https://developer.android.com/reference/android/app/AlarmManager) API. This is standard practice but does mean the delivery of the notifications/alarms are inexact and this is documented Android behaviour as per the previous link. Note that it's been reported that Samsung's implementation of Android has imposed a maximum of 500 alarms that can be scheduled via this API and exceptions can occur when going over the limit
-* iOS has a limit on how many pending notifications it allows. This is a limit imposed by iOS where it will only keep 64 notifications that will fire the soonest
-* *Known issue*: There is a known issue with handling daylight savings for scheduled notifications. This functionality may be deprecated to be replaced by another that only deals with elapsed time since epoch instead of a date.
+## ⚠ Caveats and Limitations
+Note that this plugin aims to provide abstractions for all platforms as opposed to having methods that only work on specific platforms. Each method allows passing in "platform-specifics" that contains data that is specific for customising notifications on each platform. This approach means that some scenarios may not be covered by the plugin. Feel free to fork or maintain your code for showing notifications in these situations.
```

Comment:
Just realised that perhaps this paragraph can go or tweaked as developers can now access platform-specific implementations that may have platform-specific methods. Do you want to take a stab at updating this?

Reply:
I'm not familiar with the technical changes so I will leave it for you :)

Reply:
Sure :)
Is the setting for allowing maintainers to edit this PR turned on?